packages/admin/cms-admin/src/contentScope/Controls.tsx

```diff
@@ -24,28 +25,50 @@ export function ContentScopeControls<S extends ContentScopeInterface = ContentSc
     config = {},
 }: ContentScopeControlsProps<S>): JSX.Element {
     const { scope, setScope, values } = useContentScope<S>();
+    const findLabelForScopePart = (scopePart: keyof ContentScopeInterface) => {
+        const label = values.find(({ value }) => value === scope[scopePart])?.label;
+        return label ?? capitalizeString(scope[scopePart]);
+    };
+
+    const label = Object.keys(scope)
+        .map((scopeKey: string) => findLabelForScopePart(scopeKey))
+        .join(" / ");
+
+    const groupedScopes = useMemo(() => {
+        const groupingKey = Object.keys(values[0])[0];
```

Comment:
Can `values` be an empty array? Then this would crash.

Reply:
No as far as i can see values can never be empty